content/reference/cd-aws-code-services.md

```diff
@@ -63,7 +63,7 @@ phases:
   install:
     commands:
       # pulumi
-      - curl -L https://get.pulumi.com/ | bash -s -- --version 0.16.2
+      - curl -L https://get.pulumi.com/ | sh
```

Comment:
Is there a reason we just specify `-L` instead of `-fsSL` like we mention at https://pulumi.io/quickstart/install/?

Reply:
Not really, I'll fix this up.